ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/ClientTest.java

```diff
@@ -1540,6 +1540,30 @@ public void shouldSendSessionVariablesToEndpoint() throws Exception {
     assertThat(testEndpoints.getLastSessionVariables(), is(new JsonObject().put("a", "a")));
   }
 
+  @Test
+  public void shouldSendSessionVariablesWithExecuteQuery() throws Exception {
+    // Given
+    javaClient.define("a", "a");
+
+    // When
+    javaClient.executeQuery("query;").get();
+
+    // Then
+    assertThat(testEndpoints.getLastSessionVariables(), is(new JsonObject().put("a", "a")));
+  }
+
+  @Test
+  public void shouldSendSessionVariablesWithStreamQuery() throws Exception {
+    // Given
+    javaClient.define("a", "a");
+
+    // When
+    javaClient.streamQuery("query;").get();
+
+    // Then
+    assertThat(testEndpoints.getLastSessionVariables(), is(new JsonObject().put("a", "a")));
+  }
+
```

Comment:
Let's also add unit test coverage for the additional client methods that we're supporting variable substitution for (create connector, describe source, etc). Unless I'm reading the PR wrong, create connector isn't actually covered right now. Hopefully a new unit test would catch that.